HAL/keymaster/4.1/SocketTransport.cpp

```diff
@@ -14,16 +14,19 @@
  ** See the License for the specific language governing permissions and
  ** limitations under the License.
  */
+
+#include "Transport.h"
+
 #include <sys/socket.h>
 #include <arpa/inet.h>
```

Comment:
headers not included in correct sequence

Reply:
Fixed.